openapi/saps4hana.itcm.producthierarchy/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["sap", "saps4hana", "producthhierarchy", "paid"]
```

Comment:
You can add a space for key words 
"product hierarchy"

Reply:
Fixed this